libs/protocols.liq

```diff
@@ -516,19 +526,10 @@ add_protocol(static=true,"gtts",gtts_protocol,
              doc="Generate speech synthesis using Google translate's text-to-speech API. This requires the `gtts-cli` binary.  Result may be mono.",
              syntax="gtts:Text to read")
 
-register(name="Say protocol settings","protocol.say",())
-register(name="Sox path","protocol.say.sox_path","sox")
-
 # Register the legacy say: protocol using text2wave and sox
 # @flag hidden
 def say_protocol(~rlog,~maxtime,arg) =
-  sox = get(default="sox", "protocol.say.sox_path")
-  if file.which(get(default="", "protocol.text2wave.path")) != "" then
-    [process_uri(extname="wav",uri="text2wave:#{arg}","#{sox} $(input) -c 2 $(output)")]
-  else
-    log.info(label="say_protocol", "text2wave binary not found, trying gtts.")
-    [process_uri(extname="wav",uri="gtts:#{arg}","#{sox} $(input) -c 2 $(output)")]
-  end
+  ["stereo:gtts:#{arg}", "stereo:text2wave:#{arg}"]
```

Comment:
Why two files?

Reply:
Because the first one might fail (if `gtts` is not installed) in which case it should default to the second.